kili/mutations/issue/__init__.py

```diff
@@ -0,0 +1,85 @@
+"""
+Issue mutations
+"""
+
+from typing import Literal, Optional
+from typeguard import typechecked
+from .helpers import get_issue_number
+from ...helpers import Compatible, format_result
+from .queries import GQL_APPEND_TO_ISSUES
+from ..comment.queries import GQL_APPEND_TO_COMMENTS
+from ...queries.label import QueriesLabel
+
+
+class MutationsIssue:
+    """Set of Issue mutations."""
+
+    # pylint: disable=too-few-public-methods,too-many-arguments,too-many-locals
+
+    def __init__(self, auth):
+        """Initialize the subclass.
+
+        Args:
+            auth: KiliAuth object
+        """
+        self.auth = auth
+
+    @Compatible(["v1", "v2"])
+    @typechecked
+    def append_to_issues(
+        self,
+        label_id: str,
+        project_id: str,
+        object_mid: Optional[str] = None,
+        text: Optional[str] = None,
+        type_: Literal["ISSUE", "QUESTION"] = "ISSUE",
```

Comment:
is there a better name ? `is_issue` for example ?

Reply:
The data quality team plan to add other types of issue (ex: warnings, ...) so a boolean does not seem not feasible